packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -631,21 +632,26 @@ export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
               : ''}
           </uui-button-group>
         </div>
-        <uui-color-swatches
-          id="swatches"
-          class="color-picker__swatches"
-          @change=${this.handleColorSwatchChange}>
-          ${this.swatches.map(
-            swatch =>
-              html`<uui-color-swatch
-                label="${swatch}"
-                .value=${swatch}></uui-color-swatch>`
-          )}
-        </uui-color-swatches>
+        ${this._renderSwatches()}
       </div>
     `;
   }
 
+  private _renderSwatches() {
+    if (!this.swatches) return nothing;
+    return html`<uui-color-swatches
+      id="swatches"
```

Comment:
An ID most only be present once pr. document fragment(web component, shadow dom). So we should be using a class here, as there is multiple swatch elements.

Reply:
there is only one swatches element. It is the swatch element there is multiple of 😉

Reply:
sorry my mistake. Thanks